DiscountDiscover/Views/DealCell.m

```diff
@@ -27,12 +25,15 @@ - (void)setDeal:(Deal *)deal {
         [self.dealImageView setImageWithURL:self.deal.imageUrl];
     }
     
+    [self cardStyling];
```

Comment:
This should be called just once, usually by overriding the `awakeFromNib:` method. It doesn't need to be called every time a cell is reused for a new deal (remember cells are reused on iOS as you scroll, so the same cell instance will be reused for different deals).